docs/stacks-guide-aws.md

```diff
@@ -0,0 +1,896 @@
+    name: aws-provider
+    namespace: ${INFRA_NAMESPACE}
+  reclaimPolicy: Delete
+EOF
+
+# apply the resource classes yaml to Crossplane
+kubectl apply -f "${WORK_DIR}/resource_classes.yaml"
+```
+
+So far we have been creating resources in `INFRA_NAMESPACE`, where all resources
+are to configure AWS stack with the AWS account, create a network configuration,
+and define resources classes that will satisfied the claims. Now, we will create
+an app namespace and populate it with resources that are used to let Crossplane
+know how to satisfy the claims. Let's call this namespace `myapp`
+
+```bash
+# the namespace that the app resources will be created
+APP_NAMESPACE=myapp
+# create the namespace in Crossplane
+kubectl create namespace ${APP_NAMESPACE}
+```
+
+Now we need to tell Crossplane which resource classes should be used to satisfy
+our claims in this app namespace. We will create portable classes that have
+references to cloud-specific resources we created earlier. In our claims, we can
+refer to those portable classes directly or label one as the default portable
+class to be used in claims that do not have class reference.
+
+> Portable classes are a way of referring to cloud-specific resource classes in
+> other namespaces.
+
+For example, MySQLInstanceClass is a portable class that can refer to AWS's
+RDSInstanceClass, which is a cloud-specific class.
+
+```bash
+cat > ${WORK_DIR}/portable_classes.yaml <<EOF
+---
+apiVersion: database.crossplane.io/v1alpha1
+kind: MySQLInstanceClass
+metadata:
+  name: mysql-standard
+  namespace: ${APP_NAMESPACE}
+classRef:
+  kind: RDSInstanceClass
+  apiVersion: database.aws.crossplane.io/v1alpha2
+  name: standard-mysql
+  namespace: ${INFRA_NAMESPACE}
+---
+apiVersion: compute.crossplane.io/v1alpha1
+kind: KubernetesClusterClass
+metadata:
+  name: k8s-standard
+  namespace: ${APP_NAMESPACE}
+classRef:
+  kind: EKSClusterClass
+  apiVersion: compute.aws.crossplane.io/v1alpha2
+  name: standard-cluster
+  namespace: ${INFRA_NAMESPACE}
+---
+EOF
+
+kubectl apply -f "${WORK_DIR}/portable_classes.yaml"
+```
+
+For more details about what is happening behind the scenes, read more about
+[portable claims in Crossplane][portable-claims].
+
+## Recap
+
+To recap what we've set up now in our environment:
+
+* Our provider account, both on the provider side and on the Crossplane side.
+* A Network Configuration for all instances to share.
+* An EKSClusterClass and a RDSInstanceClass with the right configuration to use
+  the mentioned networking setup.
+* A namespace for our app resources to reside with default MySQLInstanceClass
+  and KubernetesClusterClass that refer to our EKSClusterClass and
+  RDSInstanceClass.
+
+## Next Steps
+
+Next we'll set up a Crossplane App Stack and use it! Head [back over to the
+Stacks Guide document][stacks-guide-continue] so we can pick up where we left
+off.
+
+
+[AWS user]: https://docs.aws.amazon.com/mediapackage/latest/ug/setting-up-create-iam-user.html
```

Comment:
Style nitpick: I have been using lowercase and hyphens for the names for link references